blocks/cards/cards.css

```diff
@@ -176,7 +175,7 @@ main .cards.machine-parts .cards-card-body a > strong {
 main .cards.machine-parts ul > li {
```

Comment:
I assume this was called `.machine-parts` fort the engineering site, could we rename this if that is the case?